src/js/components/ServiceDetail.js

```diff
@@ -145,6 +167,33 @@ class ServiceDetail extends mixin(InternalStorageMixin, StoreMixin, TabsMixin) {
     );
   }
 
+  getRestartConfirmDialog() {
+    const {service} = this.props;
+    const {state} = this;
+
+    let message = (
+      <div className="container-pod flush-top container-pod-short-bottom">
+        <h2 className="text-danger text-align-center flush-top">Restart Service</h2>
+        <p>
+          Are you sure you want to restart <span className="emphasize">{service.getId()}</span>?
+        </p>
+        {this.getErrorMessage()}
+      </div>
+    );
+
+    return (
+      <Confirm children={message}
```

Comment:
Got a JIRA to clean up the dupe code of these modals for next sprint. Let's tackle this there, otherwise this change will cause an inconsistency in how these modals are implemented